storage/jellyfish-merkle/src/lib.rs

```diff
@@ -92,8 +92,15 @@ use node_type::{Child, Children, InternalNode, LeafNode, Node, NodeKey};
 #[cfg(any(test, feature = "fuzzing"))]
 use proptest_derive::Arbitrary;
 use std::collections::{BTreeMap, BTreeSet};
+use thiserror::Error;
 use tree_cache::TreeCache;
 
+#[derive(Error, Debug)]
+#[error("Missing root node at version {version}, probably pruned")]
```

Comment:
Probably "missing *state* root node", to be clear. State sync or API log readers see this sentence only and no stack.
Add a period (".") :-)